extensions/Skyhigh173/json.js

```diff
@@ -718,6 +718,13 @@
     json_is_valid({ json }) {
       if (typeof json != "string") {
         return false;
+      }
+      json = json.trim();
+      if (
+        (json.slice(0, 1) != "[" || json.slice(-1) != "]") &&
```

Comment:
why cant you just do json[0] or json.at(0) and json.at(-1), is there any benifit?, this isnt a review bc im just curious lol

Reply:
why did it pop up like a review :(

Reply:
There's a conflict from https://github.com/TurboWarp/extensions/pull/1596
So I copied the code from there
Perhaps I can change it after this
🤷